src/dotnet/APIView/ClientSPA/src/app/_services/signal-r/signal-r.service.ts

```diff
@@ -16,6 +17,7 @@ export class SignalRService {
   private aiReviewUpdates: Subject<AIReviewJobCompletedDto> = new Subject<AIReviewJobCompletedDto>();
   private reviewUpdates: Subject<Review> = new Subject<Review>();
   private apiRevisionUpdates: Subject<APIRevision> = new Subject<APIRevision>();
+  private siteNotifications: Subject<SiteNotificationDto> = new Subject<SiteNotificationDto>();
```

Comment:
You could potentially use the same observable for `SiteNotificationDto` and `AIReviewJobCompletedDto` since they are both notifications, Perhaps a standard model for everything notifications.

Reply:
let me see how I can combine this too, I think there are some things that makes AIReviewJobCompletedDto very specifc for that scenario as everything is populated on the notification send part

I tried to do SiteNotificationDto none specific for future usage so I will migrate AIReviewJobCompletedDto to this, if we only want one observable